std/stdio.d

```diff
@@ -1368,6 +1368,32 @@ Throws: $(D Exception) if the file is not opened.
         w.put('\n');
     }
 
+    /**
+
+    Writes its arguments in text format to the file separated pairwise by a
+    space (or custom `separator`), followed by a newline (or custom `eol`).
+
+    Throws: $(D Exception) if the file is not opened.
+            $(D ErrnoException) on an error writing to the file.
+    */
```

Comment:
`writeln` and `print` should refer to each other via `See_Also`, to reduce confusion.